plutus-conformance/src/PlutusConformance/Common.hs

```diff
@@ -38,26 +40,51 @@ data UplcEvaluationTest =
        , testDir :: FilePath
     }
 
+-- This is basically the same option as 'tasty-golden' uses, but it's not
+-- worth a dependency just to reuse the tiny datatype.
```

Comment:
Could you add an example usage here or somewhere?

Reply:
Done.